spring-kafka-test/src/main/java/org/springframework/kafka/test/rule/KafkaEmbedded.java

```diff
@@ -157,6 +159,7 @@ protected void before() throws Exception { //NOSONAR
 		for (String topic : this.topics) {
 			AdminUtils.createTopic(zkUtils, topic, this.partitionsPerTopic, this.count, props);
 		}
+		System.setProperty(SPRING_EMBEDDED_KAFKA_BROKERS, getBrokersAsString());
```

Comment:
If we are expose it into the `System Properties` in the beginning, we have to clean up it in the end.